modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -905,6 +911,30 @@
 			style["background"] = "none";
 			return style;
 		},
+		getCvaaCssForEmbedded: function(cvaaCss) {
```

Comment:
I would make a function to map between those properties, something like this:
https://github.com/jquery/jquery/blob/master/src/core.js#L293